--- origsrc/bug-buddy-2.22.0/configure.in	2008-03-10 17:42:21.000000000 -0500
+++ src/bug-buddy-2.22.0/configure.in	2008-05-29 22:37:49.609375000 -0500
@@ -86,10 +86,15 @@
 AC_SUBST(BUG_BUDDY_EDS_LIBS)
 
 
-
-AC_CHECK_LIB(elf, elf_version)
-AC_CHECK_HEADERS([libelf.h],[],[AC_MSG_ERROR([libelf.h not found.])])
-AC_CHECK_HEADERS([gelf.h],[],[AC_MSG_ERROR([gelf.h not found.])])
+case "$host_os" in
+    cygwin*|mingw*|darwin*)
+        # These platforms do not use ELF binaries
+        ;;
+    *)
+        AC_CHECK_LIB(elf, elf_version)
+        AC_CHECK_HEADERS([libelf.h gelf.h])
+        ;;
+esac
 
 AC_CONFIG_SUBDIRS(google-breakpad)
 GOOGLE_BREAKPAD_SUPPORTED_OS="no"
--- origsrc/bug-buddy-2.22.0/gnome-breakpad/Makefile.am	2007-10-13 09:42:42.000000000 -0500
+++ src/bug-buddy-2.22.0/gnome-breakpad/Makefile.am	2008-05-29 22:37:49.609375000 -0500
@@ -38,4 +38,4 @@
 
 libgnomebreakpad_la_SOURCES = gnome-breakpad.cc $(GOOGLE_BREAKPAD_CC)
 libgnomebreakpad_la_LIBADD = $(GOOGLE_BREAKPAD_LIBS) $(GNOME_BREAKPAD_LIBS)
-libgnomebreakpad_la_LDFLAGS = -rpath $(moduledir) -module -avoid-version @LDFLAGS@
+libgnomebreakpad_la_LDFLAGS = -rpath $(moduledir) -module -avoid-version -no-undefined @LDFLAGS@
--- origsrc/bug-buddy-2.22.0/gnome-breakpad/gnome-breakpad.cc	2007-10-13 09:42:42.000000000 -0500
+++ src/bug-buddy-2.22.0/gnome-breakpad/gnome-breakpad.cc	2008-05-29 22:37:49.609375000 -0500
@@ -6,8 +6,10 @@
 #include <cassert>
 #include <cstring>
 
+#if defined(HAVE_LIBELF_H) && defined(HAVE_GELF_H)
 #include <libelf.h>
 #include <gelf.h>
+#endif
 
 #include <sys/types.h>
 #include <sys/stat.h>
@@ -119,6 +121,7 @@
 }
 	
 
+#if defined(HAVE_LIBELF_H) && defined(HAVE_GELF_H)
 static gboolean
 elf_has_debug_symbols (const char *filename)
 {
@@ -171,6 +174,7 @@
        elf_end (elf);
        return false;
 }
+#endif /* HAVE_LIBELF_H && HAVE_GELF_H */
 
 
 
@@ -276,7 +280,11 @@
 	
 	memset(mypath, 0, sizeof(mypath));
 	readlink ("/proc/self/exe", mypath, sizeof(mypath));
+#if defined(HAVE_LIBELF_H) && defined(HAVE_GELF_H)
 	has_debug_symbols = elf_has_debug_symbols (mypath);
+#else
+	has_debug_symbols = FALSE;
+#endif
 
 
 	if (bugbuddy && gdb && has_debug_symbols) {
